### DISCUSSION TITLE
Why do docs recommend installing v1.59.1 instead of v1.60.3?

### DISCUSSION THREAD

**Discussion by rootulp** [golangci/golangci-lint#4995] [Q&A] [answered]
https://golangci-lint.run/welcome/install/#binaries:

> curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.59.1

But the latest release is https://github.com/golangci/golangci-lint/releases/tag/v1.60.3. 

Why do the docs recommend installing v1.59.1 instead of v1.60.3?

**Comment by ldez** [comment, marked as answer]
The link https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh is a script with no recommended version.

Maybe you are talking about https://golangci-lint.run/welcome/install/, this page recommends v1.60.3.

Could you refresh the page? Maybe your problem is just a browser cache effect.

**Reply by rootulp** [reply]
I just hard refreshed and now this command has v1.60.3 at the end so I think this is resolved.

```
# binary will be $(go env GOPATH)/bin/golangci-lint
curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.60.3

golangci-lint --version
```
